src/lib.rs

```diff
@@ -86,7 +99,7 @@ impl<'a, T: Transport> DrandClient<'a, T> {
                         &self.chain_info.public_key,
                         &beacon,
                     )
-                    .map_err(|_| DrandClientError::FailedVerification)?;
+                        .map_err(|_| DrandClientError::FailedVerification)?;
```

Comment:
is this really the linter suggestion? seems misaligned

Reply:
aye, the formatter sends it there!

Reply:
no wait... it didn't. Weirdly the RustRover formatter sends it there, but `cargo fmt` does not fuuuu